.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: test on CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.7, 3.0, 3.1, 3.2, head]
```

I think these should be strings. The yaml parser is casting `3.0` to the integer `3`, and this causes setup-ruby to install the latest Ruby 3.x version (3.2) instead of the desired 3.0.x.

```suggestion
        ruby: ["2.7", "3.0", "3.1", "3.2", "head"]
```

---

I don't know that...
Thx, I'll fix it.